src/getHumanAge.test.js

```diff
@@ -7,4 +7,27 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+  
+  test('animals ages less than 15', () => {
```

Comment:
I don't like how you name all of your tests. These are the same test cases that we've been writing all the time. Your tests case should state what expected result should be after what actions. For example - here - there are two cases. And so there should be two tests:
- should return 0 for cat and dog when they are 0 years old( but I'm not sure about testing 0 years old...)
- should return 0 for cat and dog when they are 14 years old
Please, change description of all your tests in similar way. And let's test one thing at a time everywhere!